packages/ag-charts-enterprise/src/series/radial-bar/radialBarSeries.ts

```diff
@@ -175,6 +175,10 @@ export class RadialBarSeries extends _ModuleSupport.PolarSeries<
                 ),
                 ...extraProps,
             ],
+            formatIntoColumns: false,
+            doNotFormatIntoRows: false,
+            // formatIntoColumns: true,
```

Comment:
Nit: ?

Reply:
I got this half working but need to do more fixes - so I just left this in to make it easier to re-enable. Will be fixed in the next PR or so